$MirOS: ports/multimedia/mplayer/patches/patch-libaf_af_resample_c,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- libaf/af_resample.c.orig	Sat Jan  8 21:33:44 2005
+++ libaf/af_resample.c	Thu May 19 20:32:06 2005
@@ -182,7 +182,7 @@ static int control(struct af_instance_s*
     if((s->setup & RSMP_MASK) == RSMP_LIN){
       s->pt=0LL;
       s->step=((uint64_t)n->rate<<STEPACCURACY)/(uint64_t)af->data->rate+1LL;
-      af_msg(AF_MSG_DEBUG0,"[resample] Linear interpolation step: 0x%016X.\n",
+      af_msg(AF_MSG_DEBUG0,"[resample] Linear interpolation step: 0x%016llX.\n",
 	     s->step);
       af->mul.n = af->data->rate;
       af->mul.d = n->rate;
